sources/packages/backend/libs/services/src/notifications/notification/notification.model.ts

```diff
@@ -68,11 +69,50 @@ export interface StudentNotification {
   userId: number;
 }
 
-export interface DisbursementBlockedNotificationForMinistry {
+export interface DisbursementBlockedNotification {
+  givenNames: string;
+  lastName: string;
+  email: string;
+  birthDate: string;
+  applicationNumber: string;
+}
+
+export interface ApplicationExceptionRequestNotification {
+  givenNames: string;
+  lastName: string;
+  email: string;
+  birthDate: string;
+  applicationNumber: string;
+}
+
+export interface ApplicationEditedTooManyTimesNotification {
+  givenNames: string;
+  lastName: string;
```

Comment:
I see almost all the interfaces are the same except one or 2 which does not have application number.

Please have a Base Interface and use extends.

Reply:
@dheepak-aot even though them being the same data they are targeting different template IDs that can potentially evolve differently and right now they are also processed individually. An interface would not bring a great improvement here IMO. Please let me know if this is a blocker.

Reply:
More than few interfaces (for personalization) having same properties carrying same functional details seems to be different for me. When the evolution happen we can extend the interface or have individual ones created at the point was what I felt.

Either ways, I wouldn't consider it as a blocker for this PR.